Docs/source/theory/kinetic_fluid_hybrid_model.rst

```diff
@@ -0,0 +1,141 @@
+the electrons as a fluid rather than kinetic particles. The simulation resolution
+can then be set by the ion time and length scales (commonly the ion cyclotron
+period :math:`1/\Omega_i` and ion skin depth :math:`l_i`, respectively), which
+can reduce the total simulation time drastically compared to a simulation that
+has to resolve the electron Debye length and CFL-condition based on the speed
+of light.
+
+Many authors have described variations of the kinetic ion & fluid electron model,
+generally refered to as particle-fluid hybrid or just hybrid-PIC models. The implementation
+in WarpX follows the outline from `Winske et al. (2022) <https://arxiv.org/abs/2204.01676>`_.
+This description follows mostly from that reference.
+
+Model
+-----
+
+The basic justification for the hybrid model is that the system to which it is
+applied is dominated by ion kinetics, with ions moving much slower than electrons
+and photons. In this scenario two critical approximations can be made, namely,
+neutrality (:math:`n_e=n_i`) and the Maxwell-Ampere equation can be simplified by
+neglecting the displacement current term (Darwin approximation, see `Nielson and Lewis (1976) <https://www.sciencedirect.com/science/article/abs/pii/B9780124608160500154>`_),
+giving,
+
+    .. math::
+
+        \mu_0\vec{J} = \vec{\nabla}\times\vec{B}.
+
+where :math:`\vec{J} = \vec{J}_i - \vec{J}_e` is the total electrical current i.e.
+the sum of electron and ion currents. Seeing as ions are treated in the regular
+PIC manner, the ion current, :math:`\vec{J}_i`, is known during a simulation. Therefore,
+given the magnetic field, the electron current can be calculated.
+
+If we now furher assume electrons are inertialess, the electron momentum
+equation yields,
+
+    .. math::
+
+        \frac{d(n_em_e\vec{V}_e)}{dt} = 0 = -en_e\vec{E}-\vec{J}_e\times\vec{B}-\nabla\cdot\vec{P}_e+en_e\vec{\eta}\cdot\vec{J},
+
+where :math:`\vec{V_e}=\vec{J}_e/(en_e)`, :math:`\vec{P}_e` is the electron pressure
+tensor and :math:`\vec{\eta}` is the resistivity. An expression for the electric field
+(generalized Ohm's law) can be obtained from the above as:
+
+    .. math::
+
+        \vec{E} = -\frac{1}{en_e}\left( \vec{J}_e\times\vec{B} + \nabla\cdot\vec{P}_e \right)+en_e\vec{\eta}\cdot\vec{J}.
+
+Lastly, if an electron temperature is given from which the electron pressure can
+be calculated, the model is fully constrained and can be evolved given initial
+conditions.
+
+Implementation details
+----------------------
+
+.. note::
+
+    Various verification tests of the hybrid model implementation can be found in
+    the :ref:`examples section <examples-hybrid-model>`.
+
+The kinetic-fluid hybrid extension mostly uses the same routines as the standard
+PIC algorithm with the only exception that the E-field is calculated from the
+above equation rather than it being integrated from a differential equation. The
+function `WarpX::HybridPICEvolveFields()` handles the logic to update the E&M fields
```

Comment:
Maybe, here and below, we can render these code function names with "``" instead of italic?
```suggestion
function ``WarpX::HybridPICEvolveFields()`` handles the logic to update the E&M fields
```